pyEpiabm/pyEpiabm/population.py

```diff
@@ -35,3 +35,13 @@ def setup(self) -> None:
         """
         for cell in self.cells:
             cell._setup()
+
+    def total_people(self):
+        """Returns the total number of people in the population.
+        Will obviously match the configuration parameter, but useful
+        in various sweeps to have this attached.
```

Comment:
Do we want to check this is the case somewhere / document a check?

Reply:
How would you do that? The input parameter will only come into either the toy population factory or the simulation. Guess you could do the assert there but then if it throws an error it'll be in a different class. I'll put a test in, however.